### DISCUSSION TITLE
2023/09/28/trait-in-rust

### DISCUSSION THREAD

**Discussion by giscus** [kaochenlong/kaochenlong-discuss#2] [General]
# 2023/09/28/trait-in-rust

各位看到現在，都還是在學習 Rust 這個程式語言的語法，就算知道 Stack 與 Heap 的差別，也大概了解所有權跟生命週期是怎麼運作的，但如果現在要各位動手寫點程式的話，我猜大概還是不知道怎麼下手。不用太擔心，我也是。本書前半段的內容主要是希望大家先熟悉 Rust 的設計跟語法，不求馬上就寫出什麼厲害的程式...

https://kaochenlong.com/2023/09/28/trait-in-rust.html

**Comment by tros897** [comment]
你不屬於這裡，這一段是否有錯，程式範例是fn hey(&self)，而cargo run是fn hello(&self)。

**Reply by kaochenlong** [reply]
是的，我的錯，已修正，感謝提醒 :)
